.github/workflows/deploy-s3.yml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - main
+      - create-blue-green-deployment-cadt
+    tags:
+      - 1.0.14
```

Comment:
If this tag already exists, this will never run. Do you want this to run on any future tag?

Reply:
good point.  No, I don't want it to run on any future tag, basically the version is pinned to this tag until we get rid of this bucket.